pkg/throttle/throttler.go

```diff
@@ -185,6 +192,15 @@ func (throttler *Throttler) Operate() {
 	}
 }
 
+func (throttler *Throttler) hasProxySQLStores() bool {
```

Comment:
seeing that this function is called by `collectMySQLMetrics()`, and from within a goroutine, should this be locked by a mutex? I fail to remember if  `config.Settings()` can be updated live. If not -- ignore this comment,

Reply:
@shlomi-noach good question, will confirm and potentially add a mutex 👍

Reply:
@shlomi-noach it looks like `hasProxySQLStores()` is called from `NewThrottler` only, eg: https://github.com/github/freno/pull/129/files/1e9e2d7023083f89c1b06ffbd64b8936eafbc271#diff-25a1c9a323af9f356741a7aaa4c7f15bR108

So I don't think `config.Settings()` could be changing at this stage, can you confirm?

Reply:
You're right, my mistake!

Reply:
You're right, my mistake!